iron-autogrow-textarea.html

```diff
@@ -49,8 +41,12 @@
       word-wrap: break-word;
     }
 
-    ::content textarea {
+    textarea {
+      position: relative;
```

Comment:
Or maybe expose them as a style or a mixin so that we don't have to do any shadow styling?